test_conformance/extensions/cl_khr_semaphore/test_semaphores.cpp

```diff
@@ -634,7 +674,7 @@ int test_semaphores_queries(cl_device_id deviceID, cl_context context,
     // Confirm that querying CL_SEMAPHORE_PROPERTIES_KHR returns the same
     // properties the semaphore was created with
     SEMAPHORE_PARAM_TEST_ARRAY(CL_SEMAPHORE_PROPERTIES_KHR,
-                               cl_semaphore_properties_khr, 3, sema_props);
+                               cl_semaphore_properties_khr, 5, sema_props);
```

Comment:
Earlier it was 3 and the list included the 0, shouldn't this be 6 to include the terminating property (0 here)?

Reply:
Corrected - it certainly wouldn't hurt to increase it. I just went briefly through spec and I can't spot exact info. My implementation of clGetSemaphoreInfoKHR ignores termination zero though.